### PR TITLE
#BN-18: add and view products to the wishlist

### PR DESCRIPTION
**Pull request by Honore123**
# Ticket Trello Link
[Link to Trello ticket](https://trello.com/c/oXlY8e1I/36-update-profile)
# PR Description
Allows buyers to add products to the wishlist and view the list
# Description of tasks that were expected to be completed
List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style
- [x] Validate that the buyer is logged in.
- [x] Add the item to the buyer's wishlist.
- [x] Retrieve the product details from the database.
- [x] Store the updated wishlist in the buyer's account.
# Description of tasks that were expected to be completed and that are not yet completed
- [x] None
# How this may be tested?
Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.
1. run `make build` to install all required dependencies
2. visit the `/api-docs` endpoint to access Swagger UI
3. Test wishlist endpoint `GET` and `POST`
# Screenshots (If appropriate/applicable)
Provide screenshots related to your PR in case it's appropriate and applicable
# Please check this Checklist before you submit your PR:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

**Review by uwdjany — DISMISSED** [review]
well done

File: src/config/config.json
Comment (dushimeemma) [inline src/config/config.json:1]: can we hide our sensitive data using env variables?

Comment (dushimeemma) [inline src/config/config.json:1]: - can we use env variables to hide our sensitive data?
- why do we have two configuration files?

**Review by dushimeemma — CHANGES_REQUESTED** [review]

**Review by Illustre13 — COMMENTED** [review]
Well Done Honore!

**Review by uwdjany — APPROVED** [review]

**Review by kunda4 — APPROVED** [review]
Well done

**Review by Charlesntwari — APPROVED** [review]

**Review by dushimeemma — APPROVED** [review]
Good job 🥇 🥇 🥇

**Pull request merged**